src/app/add-modules/add-modules.component.html

```diff
@@ -1,21 +1,28 @@
-<div fxLayout="column" fxLayoutGap="24px" class="add-module-container">
-  <div fxLayout="row" fxLayoutAlign="space-between center" class="crud-on-module">
-    <div class="module-text-container">
-      <p class="module-text">Modules</p>
-      <p class="module-hint">Select a module to edit, view, duplicate, change category or delete.</p>
+<div fxLayout="column" fxLayoutGap="18px" class="add-module pt-20">
+
+  <div fxLayout="row" fxLayoutAlign="space-between center" fxLayout.xs="column"
+       class="add-module__crud bg-white-color p-14">
+    <div class="add-module__text-container">
+      <p class="add-module__text font-size-20 text-light-black">Modules</p>
+      <p class="add-module__hint font-size-12 m-0 text-hint-color">Select a module to edit, view, duplicate, change
+        category or delete.</p>
     </div>
-    <div fxLayoutAlign="flex-end flex-end" class="add-module-btn">
-      <mat-icon class="add-icon">add</mat-icon>
-      <button class="add-btn" mat-button (click)="openDialog()">ADD MODULES</button>
+    <div fxLayoutAlign="end end" class="add-module__btn-container text-white-color">
+      <mat-icon class="add-module__btn-icon hand-cursor" (click)="openDialog()">add</mat-icon>
+      <button class="add-module__btn font-size-12 py-0 pr-16 pl-48 bg-primary-color" mat-button
+              (click)="openDialog()">{{'BTN_ADD_MODULES' | translate | uppercase}}</button>
     </div>
   </div>
-  <div fxLayout="row" fxLayoutAlign="space-between center" class="search-module">
-    <mat-icon class="search-icon">search</mat-icon>
-    <input type="text" placeholder="Search Modules">
-    <div fxLayout="row" class="search-right-icons">
-      <mat-icon class="view-module">view_module</mat-icon>
-      <mat-icon class="menu">menu</mat-icon>
+
+  <div fxLayout="row" fxLayoutAlign="space-between center" class="add-module__search">
+    <mat-icon class="add-module__search-icon text-sh-hint">search</mat-icon>
+    <input fxFlex="92" fxFlex.lt-md="80" class="add-module__search-bar text-sh-hint bg-white p-12 pl-44" type="text"
```

Comment:
add closing tag in input. Actually, u can use `mat-form-field` here and customize it(do it only if u have time).